fluid_dynamics/use_cases/kelvin_helmholtz_instability/source/fluid_solver_no_replace.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("FluidDynamicsApplication")
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import base class file
+from fluid_solver import FluidSolver
+from navier_stokes_solver_vmsmonolithic import NavierStokesSolverMonolithic, StabilizedFormulation
+
+class FluidSolverNoReplace(NavierStokesSolverMonolithic):
```

Comment:
you don't want to replace elements/conditions right?
Is this an option that should be available in general?

Reply:
I am not sure .. we can discuss it together with @jcotela  ... !

Reply:
by the way I realized that I put the wrong example for the results in the README.md .. I will change this and this will be gone .. !